src/components/button/Button.tsx

```diff
@@ -1,44 +1,121 @@
-import React from 'react'
-import { TouchableHighlight, ViewStyle, StyleSheet } from 'react-native'
+import React, { useRef } from 'react'
+import {
+  TouchableHighlight,
+  ViewStyle,
+  StyleSheet,
+  TextStyle,
+  View,
+  Animated
+} from 'react-native'
 import { useThemeColors } from '../../utils/theme'
 import Text from '../../components/text'
 
 import { ThemeColors, useThemedStyles } from '../../hooks/useThemedStyles'
 
 const createStyles = (themeColors: ThemeColors) =>
   StyleSheet.create({
-    button: {
+    buttonContainer: {
       backgroundColor: themeColors.primary,
-      borderRadius: 4,
+      borderRadius: 4
+    },
+    button: {
+      padding: 16,
+      borderRadius: 4
+    },
+    buttonContent: {
       display: 'flex',
-      alignItems: 'center',
-      padding: 16
+      flexDirection: 'row',
+      justifyContent: 'center',
+      alignItems: 'center'
     },
-    text: {
+    buttonText: {
       color: themeColors.staticWhite,
-      fontSize: 16
+      fontSize: 16,
+      fontWeight: 'bold'
```

Comment:
I'm not sure if `fontWeight` works with the font we are using, if it doesn't you can pass the weight to the Text component:

```<Text weight='bold'>...</Text>```